core/src/main/java/com/novoda/merlin/registerable/Registerer.java

```diff
@@ -5,79 +5,59 @@
 import com.novoda.merlin.registerable.bind.Bindable;
 import com.novoda.merlin.registerable.connection.Connectable;
 import com.novoda.merlin.registerable.disconnection.Disconnectable;
-import com.novoda.support.Logger;
 
 public class Registerer {
```

Comment:
I was tempted to change it to `Registrar`, wondered how many people would know it 😄  I'll do that.